linux_os/guide/system/accounts/enable_authselect/tests/remediable.fail.sh

```diff
@@ -1,5 +1,5 @@
 # platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel
 # packages = authselect,pam
 
-rm -f /etc/pam.d/{fingerprint-auth,smartcard-auth,system-auth}
+rm -f /etc/pam.d/{fingerprint-auth,password-auth,postlogin,smartcard-auth,system-auth}
 dnf reinstall -y pam
```

Comment:
In RHEL10 the files are now owned by `authselect-libs`.

But reinstalling them with `authselect-libs` works, but the test scenario is passing during the initial scan. I see in the commit that you say we need to remove all the files so authselect does not use the profile anymore, but it doesn't seem to be the case for RHEL10 anymore.

https://github.com/ComplianceAsCode/content/commit/50edb50a599e3d65e454d76c93e4cc62ddfe00d8